tensorboard_logger/tensorboard_logger.py

```diff
@@ -57,7 +57,7 @@ def log_value(self, name, value, step=None):
         if isinstance(value, six.string_types):
             raise TypeError('"value" should be a number, got {}'
                             .format(type(value)))
-        value = float(value)
+        # value = float(value)
```

Comment:
I think it's better to remove it instead of commenting, as it's not going to come back.